docs/targets/jira.md

```diff
@@ -0,0 +1,117 @@
+# Event Target for Jira
+
+This event Target receives [CloudEvents][ce] and invokes a Jira endpoint.
+
+## Prerequisites
+
+1. Jira instance or Atlassian cloud tenant.
+1. User API token
+
+To create the user API token at Jira:
+
+- Open the Account settings > Security > [Create and manage API Tokens][api-tokens]
+- Press `Create API token` and fill the token name.
+- Copy the API token and create a Jira token at Triggermesh.
+
+Consult the [Secrets](../guides/secrets.md) guide for more information about how to add the secrets.
+
+## Deploying an Instance of the Target
+
+Open the Bridge creation screen and add a Target of type Jira.
+
+![Adding a Jira Target](../images/jira-target/jira-target-creation.png)
+
+In the Target creation form, provide a name for the event Target, and add the following information:
+
+* **User**: the Jira user account that created the token.
+* **URL**: base URL for the Jira instance.
+
+Click the `Save` button, the console will self-navigate to the Bridge editor. Proceed by adding the remaining components to the Bridge.
+
+After submitting the bridge, and allowing some configuration time, a green check mark on the main _Bridges_ page indicates that the bridge with was successfully created.
+
+![Bridge status](../images/bridge-status-green.png)
+
+## Event Types
+
+The Jira target accepts these event types:
+
+- `com.jira.issue/create`
+
```

Comment:
Those slashes are quite inconsistent with everything we've done so far. No saying that's not a good pattern, but I'd really love to see us agreeing on _one_ consistent pattern, on only then apply it across the board. Too late to revert and discuss, I imagine?

Reply:
The target [is merged](https://github.com/triggermesh/knative-targets/pull/361).
We can open a ticket for accepted event types naming at targets or misc.

Reply:
Are we still going with `com.jira.issue.create`? I thought you had merged the target with `io.triggermesh.jira.issue.create`?